src/chainparams.cpp

```diff
@@ -39,7 +39,9 @@ class CMainParams : public CChainParams {
         // TODO generate harder genesis block
         //consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.powLimit = uint256S("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
-        consensus.nPowTargetTimespan = 14 * 24 * 60 * 60; // two weeks
+        consensus.nPowMaxAdjustDown = 16; // 16% adjustment down
+        consensus.nPowMaxAdjustUp = 8; // 8% adjustment up
+        consensus.nPowTargetTimespan = 17 * 10 * 60; // 17 blocks
```

Comment:
Reason: We're going to change the target time and don't want this to be forgotten.